README.md

```diff
@@ -169,18 +169,12 @@ ES multi-fields are for, but unfortunately those don't turn out to be quite flex
 Development
 -----------
 
-A local checkout of the frameworks repo can be shared with locally-running services (i.e. frontend applications)
-as follows:
+Whenever framework content is changed, you will need to do the following:
```

Comment:
I find `Whenever framework content is changed` a little misleading. Maybe something like `If you want to use a development branch of the frameworks repository in one of the other applications`